scylla/src/transport/connection.rs

```diff
@@ -747,6 +805,7 @@ impl Connection {
         // are carefully written in such a way that they do not hold the lock
         // across .await points. Therefore, it should not be too expensive.
         let handler_map = StdMutex::new(ResponseHandlerMap::new());
+        let (orphan_sender, orphan_receiver) = mpsc::unbounded_channel();
```

Comment:
I remember that you specifically want to use an unbounded_channel in order to be able to push to it synchronously - please add a short comment here mentioning this design decision